app/assets/stylesheets/retros.scss

```diff
@@ -50,24 +50,35 @@ html {
       }
     }
 
-    #Boards {
-      align-items: center;
+    #sections {
```

Comment:
Is there a strong reason why you are using ids and not just classes?

"The basic rule that you need to keep in mind while using classes and ids in CSS is that, id is used for single elements that appear on the page for only once (e.g. header, footer, menu), whereas class is used for single or multiple elements that appear on the page for once or more than once (e.g. paragraphs, links, buttons, input boxes). While you can also use a class for a single element, in order to get used to this distinction and use these separators as they are intended, it is better to make a habit of using classes to control the style of multiple of the same type of element."

In your case, it seems to me that it's more of a class than an id, but if you have a reason I'm good.